### DISCUSSION TITLE
Can you use a separate jamfhelper

### DISCUSSION THREAD

**Discussion by smr1619** [grahampugh/erase-install#329] [Q&A]
I was just curious if we could use a separate jamfhelper with this. This works great, but we have users that don't follow instructions and don't wait for the device to automatically restart. I just tested the full screen option, but when it finishes, there is a bout 3-4 minute delay  before it restarts. I'd like to keep the full screen message up there until it restarts, just so there is no concern of the user restarting.  Thanks again

**Comment by grahampugh** [comment]
JamfHelper is being replaced with swiftDialog in the latest version. There won't be any significant changes to v27.x. If I'm lucky, using swiftDialog will improve the end sequence.
